### ISSUE TITLE
Hello again is it possible to update healium for shadowlands?

### ISSUE THREAD

**Issue by Gal10** [Toludin/Healium#48]
It seems that the writer of healium came back last year once to update healium for Classic only. For Bfa nothing was done and now in Shadowlands it was never updated too.

Please can you help all Healium healers again and make it viable for shadowlands. we cant load it on prepatch. Please help us. we are very sad.

**Comment by Zitania** [comment]
I second what Gal10 says.. I am a healer in Shadowlands and I am very sad :(

**Comment by Amenelita** [comment]
Yes please I need Healium for the prepatch and Shadowlands. Please help us.

**Comment by Gal10** [comment]
Dedicated to Toludin:

If you can hear our voice
Helps us heal and will rejoice
Move your heart just a little
Solve for us this mysterious riddle
Healers need a glorious leader :D

**Comment by hannahxxashley** [comment]
Please help us!! I love the healium addon so much T_T I can’t be without it.

**Comment by Beawting** [comment]
Same here, i love to play with healium, and now after pre-patch, it doesn't work :-(  hope someone has time to fix the problem.

**Comment by hellkatt6** [comment]
please please please update for Shadowlands, we are begging you. I have been using this addon for 7 and absolutely love it, its the only one that makes sense to me. Please Obi Wan, help us!

**Comment by jcle4** [comment]
healium is by far my favourite healing addon and i have used it for 10 years. I would even donate to the cause! <3

**Comment by cindynico** [comment]
> Tómenlo con calma, muchos de los addons actualmente están rotos, incluyendo el Healium. Lo que de momento observo es:
> 
> El botón en el minimapa ha dejado de funcionar, quizá por obsolescencia de las funciones, requiere re-programar esa parte.
> 
> ==============================================
> 
> Take it easy, many of the addons are currently broken, including the Healium. What I am observing at the moment is:
> 
> The button on the minimap has stopped working, maybe because of obsolescence of the functions, it requires re-programming that part.

Will there be a fix any time soon?

**Comment by Hatebagel** [comment]
As a former programmer I thought I could look into what changes were made in WoW 9.x that would cause Healium to stop working.
A major fault appears to be changes in how frame borders and background can be colored.
For instant, SetBackdropBorderColor no longer functions which seems to be the big error generator when trying to load Healium.
Having never written an addon before, I'd have to really start from the beginning to find out how to write in a workaround for these changes.
However, if any programmers who are familiar with WoW addons find themselves here, I'm hoping this will help give them a legup on what needs to be done to update this addon for Shadowlands.
